### PR TITLE
Documentation on multi-filesystem support

### PR DESCRIPTION
**Pull request by nareddyt**
# Overview

Closes pnpm/pnpm#1515

Added docs to clarify how pnpm usage on multi-filesystem computers.

This information was deduced based on pnpm/pnpm#712, related issues and PRs, and the current code located [here](https://github.com/pnpm/pnpm/blob/36b53268f09796a6393f00ca38a0c4b68d3055c2/packages/package-store/src/storeController/createImportPackage.ts#L56)

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ Direct symlinking to the global store would work with Node's `--preserve-symlink
...
+The package store should be on the same disk as installations. 
+Otherwise packages will be copied, not linked. 
+This is due to a OS limitation in hard-linking. See [Issue #712](https://github.com/pnpm/pnpm/issues/712) for more details.
+
+For example, if you run `pnpm install` on disk `D:`, then the pnpm store must be on disk `D:`.
+If the pnpm store is located on disk `C:`, then all required packages will be directly copied to the project location.
+This severely reduces the benefits of pnpm.
+
+For clarity, please specify the store location manually via [the store config](configuring.md).
+If no store path is specified, then the current filesystem is used.
```
Comment (zkochan) [inline docs/faq.md]: The way it currently works. If no store path is set. pnpm will automatically create a store on the same drive.

So if someone runs installation on disk `D:`, store will be created in `D:\.pnpm-store`. If install is run on disk C, store folder will be created at `C:\.pnpm-store`. If disk C has a home dir, store will be created in the home dir

**Comment by nareddyt** [comment]
Hey, just updated. Let me know if my understanding is correct here or if the FAQ is too wordy.

File: docs/faq.md [hunk 2]
```diff
@@ -41,6 +41,30 @@ Direct symlinking to the global store would work with Node's `--preserve-symlink
...
+
+The package store should be on the same disk as installations. 
+Otherwise packages will be copied, not linked. 
+This is due to a OS limitation in hard-linking. See [Issue #712](https://github.com/pnpm/pnpm/issues/712) for more details.
+
+pnpm functions differently based on the 2 cases below:
+
+### Store path is specified
+
+If the store path is specified via [the store config](configuring.md), then copying occurs between the store and the project.
```
Comment (zkochan) [inline docs/faq.md]: "then copying occurs between the store and any projects that are from a different disk"

**Comment by zkochan** [comment]
thanks!
